docs/development/xenial_support.rst

```diff
@@ -13,22 +13,42 @@ based on Xenial.
 Running the Xenial dev env
 --------------------------
 
+The Xenial staging environment uses libvirt as a VM provider. If you haven't 
+already done so, you should follow the instructions here: :ref:`libvirt_provider`.
+
+Once the libvirt Vagrant provider is available, you'll need a libvirt-format Xenial
+base image. To set one up, run the following commands.
+
+.. code:: sh
+
+   vagrant box add bento/ubuntu-16.04  # choose the virtualbox option
+   vagrant mutate bento/ubuntu-16.04 mutate libvirt
+
+
 Due to packaging logic changes, you'll need to build the Debian packages
 with overrides enabled for Xenial support. Then run the Xenial scenario.
 
-.. code::
+.. code:: sh
 
    make build-debs-xenial
    make staging-xenial
 
-The VMs will be available. Further debugging likely required; you can
+
+The VMs will be available.  You can
 log into the machines with e.g.:
 
-.. code::
+.. code:: sh
 
    molecule login -s libvirt-staging-xenial -h app-staging
 
-Depending on the error, simply re-running the ``make staging-xenial`` target
+To run the testinfra tests against the Xenial enviroment, you can use the command:
+
+.. code:: sh
+
+  molecule verify -s libbirt-staging-xenial
```

Comment:
Nit: typo in `libvirt`. It may be worthwhile wrapping these, so that e.g. `make staging-verify` uses the same logic as the converge script, and chooses the right provider for the host system (respecting the VAGRANT_DEFAULT_PROVIDER env var, as it does now). Never implemented that, since I'm not sure it would actually be useful to folks.

Reply:
That would simplify the docs a lot, and give devs less to think about, but I'm gonna leave it for another PR.